queries/haskell/highlights.scm

```diff
@@ -150,13 +176,77 @@
 
 ; True or False
 ((constructor) @boolean (#any-of? @boolean "True" "False"))
-
+; otherwise (= True)
+((variable) @boolean (#set! "priority" 101)
```

Comment:
Is it a must for this to have priorities? Can't it be moved down a bit for nvim queries  since ours is "last wins"

Reply:
You're right, this one (and the equivalent ones) are redundant.
Not sure why they weren't working for me yesterday.